modules/500-upmeter/images/upmeter/src/pkg/probe/group_deckhouse.go

```diff
@@ -27,12 +30,76 @@ import (
 	"d8.io/upmeter/pkg/probe/run"
 )
 
+type probeConfig struct {
+	WindowSize                  int
+	FreezeThreshold             time.Duration
+	AllowedTasksPerTimeInterval float64
+	TaskGrowthThreshold         float64
+}
+
+var (
+	config          probeConfig
+	probeConfigOnce sync.Once
+)
+
+func parseConfigFromEnv(logger *logrus.Logger, period time.Duration) probeConfig {
```

Comment:
It belongs here https://github.com/deckhouse/deckhouse/blob/83b34b7c59e74dca318c2b25134adbdf882310a2/modules/500-upmeter/images/upmeter/src/cmd/upmeter/parse_args.go#L148-L199

Reply:
moved to parse_args